README.md

```diff
@@ -8,6 +8,19 @@ This project is developed and maintained by the [Embedded Linux team][team].
 
 ## [Documentation](https://docs.rs/linux-embedded-hal)
 
+## GPIO character device
+
+Since Linux kernel v4.4 the use of sysfs GPIO was deprecated and replaced by the character device GPIO.
+See [gpio-cdev documentation](https://github.com/rust-embedded/gpio-cdev#sysfs-gpio-vs-gpio-character-device) for details.
+
+This crate includes feature flag `gpio_cdev` that exposes `CdevPin` as wrapper around `LineHandle` from  [gpio-cdev](https://crates.io/crates/gpio-cdev).
```

Comment:
maybe we could also have a note on dropping linehandles here and in the rustdocs for the CdevPin?

(relates to: https://github.com/rust-embedded/gpio-cdev/issues/29)

Reply:
also if you could also add the notes to the changelog that'd be excellent, otherwise one of us can do it before releasing.

(thanks again for all your work / sorry for the back and forth!)

Reply:
Done, I was happy to help, thanks. Is there anything else I could do for the project?